scripts/skills/actives/rf_sling_smoke_bomb_skill.nut

```diff
@@ -0,0 +1,10 @@
+this.rf_sling_smoke_bomb_skill <- ::inherit("scripts/skills/actives/throw_smoke_bomb_skill", {
+	function create()
+	{
+        this.throw_smoke_bomb_skill.create();
+        this.m.Icon = "skills/rf_sling_smoke_bomb.png";
```

Comment:
We are going with the convention of appending `_skill` to the icons of active skills. So, please rename this to `rf_sling_smoke_bomb_skill.png`. Similarly for the black and white version: `rf_sling_smoke_bomb_skill_sw.png`. Please update for other skills too.